harness/determined/cli/user.py

```diff
@@ -445,42 +426,43 @@ def update_token(args: argparse.Namespace) -> None:
         ]),
         cli.Cmd("token", None, "manage access tokens", [
             cli.Cmd("describe", describe_token, "describe token info", [
-                cli.Arg("username", nargs=argparse.ONE_OR_MORE, default=None,
-                        help="name of user to describe token"),
+                cli.Arg("token_id", type=int, nargs=argparse.ONE_OR_MORE, default=None,
+                        help="describe given access token id"),
```

Comment:
nit (style): Should this help be more focused on what the `token_id` parameter is for? Something maybe like:
"token id(s) specifying access tokens to describe"

Reply:
Or maybe we should make sure it's clear that this could be multiple values as well, so if we run with the current help string maybe we could do something like:
"describe given access token id(s)"